contracts/strategies/convex/ConvexStrategyEURT_USDMainnet.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity 0.5.16;
+
+import "./base/ConvexStrategyUL.sol";
+
+contract ConvexStrategyEURT_USDMainnet is ConvexStrategyUL {
+
+  address public eurs_usd_unused; // just a differentiator for the bytecode
+
+  constructor() public {}
+
+  function initializeStrategy(
+    address _storage,
+    address _vault
+  ) public initializer {
+    address underlying = address(0x3b6831c0077a1e44ED0a21841C3bC4dC11bCE833); // Info -> LP Token address
+    address rewardPool = address(0xD2B756Af4E345A8657C0656C148aDCD3000C97A4); // Info -> Rewards contract address
+    address crv = address(0xD533a949740bb3306d119CC777fa900bA034cd52);
+    address cvx = address(0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B);
+    address eurt = address(0xC581b735A1688071A1746c968e0798D642EDE491);
```

Comment:
You might have meant to use DAI to deposit into this pool instead of the EURT one